### DISCUSSION TITLE
IDF V5 - Arduino Core Update

### DISCUSSION THREAD

**Discussion by DerPicknicker** [espressif/arduino-esp32#7772] [Q&A] [answered]
Hi, 

just a Short question: When is it planned to update the IDF Version to 5? I need some fixes from the newer Version. I don’t want to change to Esp-IDF.. 

Do you have any timeframe to update the Arduino Core to the latest IDF-Version? 

Or do you have any alternative for me? I saw the Arduino Lib-Builder is that a solution for me?

best regards!

**Comment by me-no-dev** [comment, marked as answer]
We are working on it in the `esp-idf-v5.1-libs` branch, but it's too early to use. No timeframe yet

**Reply by DerPicknicker** [reply]
> We are working on it in the `esp-idf-v5.1-libs` branch, but it's too early to use. No timeframe yet

Thank you ;) .. Could you explain what the Arduino-Lib-Builder is used for?

**Reply by me-no-dev** [reply]
@DerPicknicker https://docs.espressif.com/projects/arduino-esp32/en/latest/lib_builder.html
